client/src/app/project-create-edit/components/project-metadata-form/layout.ts

```diff
@@ -33,6 +33,7 @@ const getFullLayout = () => ({
             'project.content.insdc_project_accessions',
             'project.content.insdc_study_accessions',
             'project.content.ega_accessions',
+            'project.content.dbgap_accessions',
```

Comment:
I might have not been clear, or have not understood your comment.
I mean, that instead of repeating the list of accession keys here, to reuse the list defined in the `accessionFields` property in `accession-field-group.component.ts`

Reply:
Ah I see, I will see what I can do